supertokens_python/recipe/passwordless/utils.py

```diff
@@ -146,7 +146,13 @@ def __init__(self,
                  ):
         super().__init__('EMAIL_OR_PHONE')
         self.email_delivery = email_delivery
-        self.create_and_send_custom_email = create_and_send_custom_email
+
+        if create_and_send_custom_email is None:
+            warn("create_and_send_custom_email is depricated. Please use email delivery config instead")
```

Comment:
This is wrong. First, we don't do deprecation like this. And second, you are doing this if the user has NOT given `create_and_send_custom_email`

Reply:
Fixed.